docs/resources/fex_profile.md

```diff
@@ -7,40 +10,118 @@ description: |-
   Manages ACI FEX Profile
 ---
 
-# aci_fex_profile
+# aci_fex_profile #
 
 Manages ACI FEX Profile
 
-## Example Usage
+
+
+## API Information ##
+
+* Class: [infraFexP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraFexP/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Format: `uni/infra/fexprof-{name}`
+
+## GUI Information ##
+
+* Location: `Fabric -> Access Policies -> Interfaces -> Leaf Interfaces -> Profiles -> Fex Interfaces`
+
+## Example Usage ##
+
+The configuration snippet below creates a FEX Profile with only required attributes.
 
 ```hcl
 
 resource "aci_fex_profile" "example" {
-  name        = "fex_prof"
-  annotation  = "example"
-  name_alias  = "example"
-  description = "from terraform"
+  name = "test_name"
 }
 
 ```
+The configuration snippet below shows all possible attributes of the FEX Profile.
 
-## Argument Reference
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
 
-- `name` - (Required) The FEX profile name.
-- `annotation` - (Optional) Specifies the annotation of the FEX profile name.
-- `description` - (Optional) Specifies the description of the FEX profile name.
-- `name_alias` - (Optional) Specifies the alias name of the FEX profile name.
+resource "aci_fex_profile" "full_example" {
+  annotation  = "annotation"
+  description = "description_1"
+  name        = "test_name"
+  name_alias  = "name_alias_1"
+  owner_key   = "owner_key_1"
+  owner_tag   = "owner_tag_1"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
 
-## Attribute Reference
+All examples for the FEX Profile resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_fex_profile) folder.
 
-The only attribute that this resource exports is the `id`, which is set to the
-Dn of the FEX Profile.
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the FEX Profile object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the FEX Profile object.
+
+### Optional ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
```

Comment:
Do we need a parent_dn for this resource and the others if "uni/infra" is the only possible option?